test/unit/qme/map/measure_calculation_job_test.rb

```diff
@@ -48,8 +48,7 @@ def test_perform_recalculate
   def test_perform_no_recalculate
     options = {
       'measure_id' => '2E679CD2-3FEC-4A75-A75A-61403E5EFEE8',
-      'effective_date' => Time.gm(2011, 1, 15).to_i,
-      'status.state' => 'complete'
```

Comment:
Hm, that's strange. We specifically added this test to solve a bug earlier this year. https://github.com/q-centrix/quality-measure-engine/commit/29e700de74428833c2d4c94309d4b2b97c4f383c#diff-619e38178e285e46efab68c7a8d5493f.